docs/instruction_for_testing.md

```diff
@@ -0,0 +1,82 @@
+
+At a high level, you will need to:
+
+1. Deploy 4 *additional* machines that will be dedicated to running the integration tests (Windows, RHEL 7, RHEL 8, macOS - if available)
+1. Copy over the `integrations_tests_runner` executable to the new machines
+1. Copy over the artifacts referenced in the integration tests to the new machines. These artifacts live under the `test` directory so just copy the whole `test` directory.
+1. Run the executable
+
+The following is an example of how you would run the `integrations_tests_runner` executable:
+
+```
+./integrations_tests_runner \
+  --url https://conjur-server \
+  --account someaccount \
+  --login somelogin \
+  --password Myp@SS0rdsS1! \
+  --files-folder test
+```
+
+Where `url`, `account`, `login`, `password` detail the Conjur server-specific information you previously deployed. Depending on the type of Conjur server you deployed, two tests should fail. For example, if you have configured a Conjur server with a verifiable CA-signed certificate, then `test_https_cli_fails_if_cert_is_bad` and `test_https_cli_fails_if_cert_is_not_provided` should fail. "Why" you may ask? See the developer note above each test or see the issue created [here](https://github.com/cyberark/conjur-api-python3/issues/209). For all other flows, these tests should pass.
+
+Congrats! You ran project's integration tests.
+
+## Running tests with configurable environment
+
+As part of our test suite, we also want to test the CLI against different types of Conjur server environments. We want to validate the flow when the CLI makes a request to a server and the certificate configured is either 1. a known, verifiable CA-signed certificate or 2. a non-verifiable CA-signed certificate.
+
+Python comes with a list of trusted CAs. Every certificate signed by those CAs will be considered trusted certificate. Those CAs are located in a file called `cacerts.pem`. To get the path to this file, run in `python3 -c "import certifi; print(certifi.where())"`
+
+At a high level, you will need to:
+
+1. Pick a platform to run the additional test flow (Hint: macOS should be the easiest)
+1. Complete the walk-through doc  (TODO ADD LINK) for configuring a certificate starting at the *Build CLI with your CA included* section.
+1. Once you have built the `integrations_tests_runner` with a verifiable CA-signed certificate, run the `integrations_tests_runner` executable one last time. The two tests mentioned in the previous section should fail.
+
+Note that you should remove the `build` and `dist` directories to make sure you build a fresh executable!
+
+## Miscellaneous tests
+
+There are a couple of manual checks that need to be done in additional to the regular integration tests.
+
+You should compress the *signed* Windows CLI executables (not the `integrations_test_runner`!) and run it on a Windows machine. 
+
+This will verify two things:
+
+1. The executable doesn't get corrupted during compression/decompression 
```

Comment:
TODO - The final deliverable is a `zip` so this ensures that the executable doesn't get corrupted during compression/decompression